public/scripts/views/forms/custom-id.js

```diff
@@ -5,9 +5,12 @@
     controller: function (element, sdk, console, window) {
       let prevData = "";
       let idType = element.getAttribute('data-id-type');
+      let disableSwitch = element.getAttribute('data-disable-switch');
 
       const div = window.document.createElement("div");
-      div.className = "input-copy";
+      if(disableSwitch !== "true") {
+        div.className = "input-copy";
```

Comment:
Why do we need the copy icon?

Reply:
not the copy icon, its the css class for styling the end button, I'm using the same class, if  the button is not required, this class is not required

Reply:
Ohh, OK thanks for clarifying.